components/tools/OmeroJava/test/integration/PermissionsTest.java

```diff
@@ -114,4 +128,45 @@ public void testRootCanPermissionsByProjection() throws Exception {
         Assert.assertTrue(((RBool) projectPermsMap.get("canChgrp")).getValue());
         Assert.assertTrue(((RBool) projectPermsMap.get("canChown")).getValue());
     }
+
+    /**
+     * Test that an instance of the given class can be annotated only if the instance's {@link Permissions#canAnnotate()} is true.
+     * @param annotateeClass the class of which to try annotating an instance
```

Comment:
is it intended ``annotatee``?

Reply:
"Annotatee" seemed the best succinct term for "annotated thing", i.e., the target of the annotation. I'm [not the first](http://www.iis.sinica.edu.tw/~swc/pub/information_leakage_socialnets.html) to coin it but admittedly it'll take a few more of us to get it into dictionaries!